app/views/layouts/membership_pdf.html.erb

```diff
@@ -13,7 +13,17 @@
 <div>
   <%= yield %>
 </div>
-Budapest, 20__. ____________ __.
+<div class="signature">
+  <div class="row">
+    <div class= "col">
+      Budapest, 20__. ____________ __.
+    </div>
+    <div class="col">
+      <hr>
```

Comment:
There is some problem with the line for the signature, it breaks. Please check and fix it, if possible.

Reply:
I know, but I didn't find why. And it is just a fly in the ointment and we have more important tasks. 
So I wrote to a Trello card